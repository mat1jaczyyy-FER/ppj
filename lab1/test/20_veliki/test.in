
x
1
abcd
xyz5
5xyz
12346546756867898707851234123451089766644
x=5
7*3
1-3/5+7
a+b*(12+43/4)*12-(23*(2*(4)))
(((((((a+b)-c)+d)/11)*34)*1234)-345)
a + b
a +    b +c+d
  
a = x+y // moo
// komentar
a = x+y // x + y
x = 123 // var
abc = 765+123/ (123
-123
+32)

za a od 3 do 5
  xy = abc+32 // nop
azabc = 765+123/ (123
-123
+32)

za a od 3 do 5
  xy = abc+32 // nop
az
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
za a od 3 do 5
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
  xy = abc+32 // nop
az
za n od 3 do 321
  x = x - 1
az

// komentar
// y+z
za i od -x do x
  x = x*x
// az
b = 12 // &^#$ bilo sto
// prazan redak
// broji retke ispravno
x + 3
   

    

    123 3	 abc312 	abc 	123 +x 	-x


		
+e
+       f
