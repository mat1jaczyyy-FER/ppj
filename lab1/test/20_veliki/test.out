IDN 2 x
BROJ 3 1
IDN 4 abcd
IDN 5 xyz5
BROJ 6 5
IDN 6 xyz
BROJ 7 12346546756867898707851234123451089766644
IDN 8 x
OP_PRIDRUZI 8 =
BROJ 8 5
BROJ 9 7
OP_PUTA 9 *
BROJ 9 3
BROJ 10 1
OP_MINUS 10 -
BROJ 10 3
OP_DIJELI 10 /
BROJ 10 5
OP_PLUS 10 +
BROJ 10 7
IDN 11 a
OP_PLUS 11 +
IDN 11 b
OP_PUTA 11 *
L_ZAGRADA 11 (
BROJ 11 12
OP_PLUS 11 +
BROJ 11 43
OP_DIJELI 11 /
BROJ 11 4
D_ZAGRADA 11 )
OP_PUTA 11 *
BROJ 11 12
OP_MINUS 11 -
L_ZAGRADA 11 (
BROJ 11 23
OP_PUTA 11 *
L_ZAGRADA 11 (
BROJ 11 2
OP_PUTA 11 *
L_ZAGRADA 11 (
BROJ 11 4
D_ZAGRADA 11 )
D_ZAGRADA 11 )
D_ZAGRADA 11 )
L_ZAGRADA 12 (
L_ZAGRADA 12 (
L_ZAGRADA 12 (
L_ZAGRADA 12 (
L_ZAGRADA 12 (
L_ZAGRADA 12 (
L_ZAGRADA 12 (
IDN 12 a
OP_PLUS 12 +
IDN 12 b
D_ZAGRADA 12 )
OP_MINUS 12 -
IDN 12 c
D_ZAGRADA 12 )
OP_PLUS 12 +
IDN 12 d
D_ZAGRADA 12 )
OP_DIJELI 12 /
BROJ 12 11
D_ZAGRADA 12 )
OP_PUTA 12 *
BROJ 12 34
D_ZAGRADA 12 )
OP_PUTA 12 *
BROJ 12 1234
D_ZAGRADA 12 )
OP_MINUS 12 -
BROJ 12 345
D_ZAGRADA 12 )
IDN 13 a
OP_PLUS 13 +
IDN 13 b
IDN 14 a
OP_PLUS 14 +
IDN 14 b
OP_PLUS 14 +
IDN 14 c
OP_PLUS 14 +
IDN 14 d
IDN 16 a
OP_PRIDRUZI 16 =
IDN 16 x
OP_PLUS 16 +
IDN 16 y
IDN 18 a
OP_PRIDRUZI 18 =
IDN 18 x
OP_PLUS 18 +
IDN 18 y
IDN 19 x
OP_PRIDRUZI 19 =
BROJ 19 123
IDN 20 abc
OP_PRIDRUZI 20 =
BROJ 20 765
OP_PLUS 20 +
BROJ 20 123
OP_DIJELI 20 /
L_ZAGRADA 20 (
BROJ 20 123
OP_MINUS 21 -
BROJ 21 123
OP_PLUS 22 +
BROJ 22 32
D_ZAGRADA 22 )
KR_ZA 24 za
IDN 24 a
KR_OD 24 od
BROJ 24 3
KR_DO 24 do
BROJ 24 5
IDN 25 xy
OP_PRIDRUZI 25 =
IDN 25 abc
OP_PLUS 25 +
BROJ 25 32
IDN 26 azabc
OP_PRIDRUZI 26 =
BROJ 26 765
OP_PLUS 26 +
BROJ 26 123
OP_DIJELI 26 /
L_ZAGRADA 26 (
BROJ 26 123
OP_MINUS 27 -
BROJ 27 123
OP_PLUS 28 +
BROJ 28 32
D_ZAGRADA 28 )
KR_ZA 30 za
IDN 30 a
KR_OD 30 od
BROJ 30 3
KR_DO 30 do
BROJ 30 5
IDN 31 xy
OP_PRIDRUZI 31 =
IDN 31 abc
OP_PLUS 31 +
BROJ 31 32
KR_AZ 32 az
KR_ZA 33 za
IDN 33 a
KR_OD 33 od
BROJ 33 3
KR_DO 33 do
BROJ 33 5
KR_ZA 34 za
IDN 34 a
KR_OD 34 od
BROJ 34 3
KR_DO 34 do
BROJ 34 5
KR_ZA 35 za
IDN 35 a
KR_OD 35 od
BROJ 35 3
KR_DO 35 do
BROJ 35 5
KR_ZA 36 za
IDN 36 a
KR_OD 36 od
BROJ 36 3
KR_DO 36 do
BROJ 36 5
KR_ZA 37 za
IDN 37 a
KR_OD 37 od
BROJ 37 3
KR_DO 37 do
BROJ 37 5
KR_ZA 38 za
IDN 38 a
KR_OD 38 od
BROJ 38 3
KR_DO 38 do
BROJ 38 5
KR_ZA 39 za
IDN 39 a
KR_OD 39 od
BROJ 39 3
KR_DO 39 do
BROJ 39 5
KR_ZA 40 za
IDN 40 a
KR_OD 40 od
BROJ 40 3
KR_DO 40 do
BROJ 40 5
KR_ZA 41 za
IDN 41 a
KR_OD 41 od
BROJ 41 3
KR_DO 41 do
BROJ 41 5
KR_ZA 42 za
IDN 42 a
KR_OD 42 od
BROJ 42 3
KR_DO 42 do
BROJ 42 5
KR_ZA 43 za
IDN 43 a
KR_OD 43 od
BROJ 43 3
KR_DO 43 do
BROJ 43 5
KR_ZA 44 za
IDN 44 a
KR_OD 44 od
BROJ 44 3
KR_DO 44 do
BROJ 44 5
IDN 45 xy
OP_PRIDRUZI 45 =
IDN 45 abc
OP_PLUS 45 +
BROJ 45 32
KR_AZ 46 az
IDN 47 xy
OP_PRIDRUZI 47 =
IDN 47 abc
OP_PLUS 47 +
BROJ 47 32
KR_AZ 48 az
IDN 49 xy
OP_PRIDRUZI 49 =
IDN 49 abc
OP_PLUS 49 +
BROJ 49 32
KR_AZ 50 az
IDN 51 xy
OP_PRIDRUZI 51 =
IDN 51 abc
OP_PLUS 51 +
BROJ 51 32
KR_AZ 52 az
IDN 53 xy
OP_PRIDRUZI 53 =
IDN 53 abc
OP_PLUS 53 +
BROJ 53 32
KR_AZ 54 az
IDN 55 xy
OP_PRIDRUZI 55 =
IDN 55 abc
OP_PLUS 55 +
BROJ 55 32
KR_AZ 56 az
IDN 57 xy
OP_PRIDRUZI 57 =
IDN 57 abc
OP_PLUS 57 +
BROJ 57 32
KR_AZ 58 az
IDN 59 xy
OP_PRIDRUZI 59 =
IDN 59 abc
OP_PLUS 59 +
BROJ 59 32
KR_AZ 60 az
IDN 61 xy
OP_PRIDRUZI 61 =
IDN 61 abc
OP_PLUS 61 +
BROJ 61 32
KR_AZ 62 az
IDN 63 xy
OP_PRIDRUZI 63 =
IDN 63 abc
OP_PLUS 63 +
BROJ 63 32
KR_AZ 64 az
IDN 65 xy
OP_PRIDRUZI 65 =
IDN 65 abc
OP_PLUS 65 +
BROJ 65 32
KR_AZ 66 az
IDN 67 xy
OP_PRIDRUZI 67 =
IDN 67 abc
OP_PLUS 67 +
BROJ 67 32
KR_AZ 68 az
KR_ZA 69 za
IDN 69 n
KR_OD 69 od
BROJ 69 3
KR_DO 69 do
BROJ 69 321
IDN 70 x
OP_PRIDRUZI 70 =
IDN 70 x
OP_MINUS 70 -
BROJ 70 1
KR_AZ 71 az
KR_ZA 75 za
IDN 75 i
KR_OD 75 od
OP_MINUS 75 -
IDN 75 x
KR_DO 75 do
IDN 75 x
IDN 76 x
OP_PRIDRUZI 76 =
IDN 76 x
OP_PUTA 76 *
IDN 76 x
IDN 78 b
OP_PRIDRUZI 78 =
BROJ 78 12
IDN 81 x
OP_PLUS 81 +
BROJ 81 3
BROJ 86 123
BROJ 86 3
IDN 86 abc312
IDN 86 abc
BROJ 86 123
OP_PLUS 86 +
IDN 86 x
OP_MINUS 86 -
IDN 86 x
OP_PLUS 90 +
IDN 90 e
OP_PLUS 91 +
IDN 91 f
